### DISCUSSION TITLE
`useFakeTimers` not exported from `@storybook/jest`

### DISCUSSION THREAD

**Discussion by assaf-aviram** [storybookjs/storybook#25867] [Help] [answered]
### Summary

the `jest` object I get from '@storybook/jest' doesn't seem to have the `useFakeTimers` function.
Is there a plan to add it, or is there a better way to fake timers in storybook tests?

**Comment by valentinpalkovic** [comment]
@storybook/jest isn’t maintained anymore. Please use @storybook/test instead

**Reply by assaf-aviram** [reply]
thanks. does `@storybook/test` export fake timers?

**Reply by valentinpalkovic** [reply]
@kasperpeulen might have more context.

**Reply by kasperpeulen** [reply, marked as answer]
Not yet, but we are planning to in the future @assaf-aviram !

**Reply by kasperpeulen** [reply]
Feel free to contribute

**Reply by zgrybus** [reply]
@kasperpeulen Any updates? I really need this utility

**Reply by kasperpeulen** [reply]
For now, I would suggest:
https://www.npmjs.com/package/mockdate

Or using sinon directly:
https://github.com/sinonjs/fake-timers

We will do this at some point. But not in the near future.

**Comment by valneras** [comment]
Alternative implemented: do a setTimeout async like: new Promise((resolve) => setTimeout(resolve));
